### PR TITLE
chore: improve cloud.Function docs

### PR DESCRIPTION
**Pull request by Chriscbr**
Add docs that were missing from https://github.com/winglang/wing/pull/6790

## Checklist

- [x] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [x] Description explains motivation and solution
- [ ] Tests added (always)
- [x] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

File: docs/docs/04-standard-library/cloud/function.md [hunk 1]
```diff
@@ -148,6 +152,52 @@ let f = new cloud.Function(inflight () => {
 
 The `context()` method returns `nil` when ran on non-AWS targets.
 
+#### Adding Lambda layers
+
+To add a Lambda layer to the function, you can use the `aws.Function` class as shown below.
```
Comment (yoav-steinberg) [inline docs/docs/04-standard-library/cloud/function.md:157]:
```suggestion
To add a [Lambda layer](https://docs.aws.amazon.com/lambda/latest/dg/chapter-layers.html) to the function, you can use the `aws.Function` class as shown below.
```

**Review by eladb — APPROVED** [review]
@boyney123 take a look

File: libs/wingsdk/src/cloud/function.md
Comment (yoav-steinberg) [inline libs/wingsdk/src/cloud/function.md:1]: This file seems like a dup of _docs/docs/04-standard-library/cloud/function.md_. Do we need to keep them in sync?

Comment (Chriscbr) [reply libs/wingsdk/src/cloud/function.md:1]: At the moment I think so, though the process is automated (`pnpm turbo compile` in the repo will automatically copy file from `libs/wingsdk/` to `docs/`). IIRC the original intent was to improve the contributing experiecne by keeping these docs close to the source code

File: libs/wingsdk/src/cloud/function.md [hunk 2]
```diff
@@ -148,6 +152,52 @@ let f = new cloud.Function(inflight () => {
 
 The `context()` method returns `nil` when ran on non-AWS targets.
 
+#### Adding Lambda layers
+
+To add a Lambda layer to the function, you can use the `aws.Function` class as shown below.
```
Comment (Chriscbr) [inline libs/wingsdk/src/cloud/function.md:157]:
```suggestion
To add a [Lambda layer](https://docs.aws.amazon.com/lambda/latest/dg/chapter-layers.html) to the function, you can use the `aws.Function` class as shown below.
```

**Pull request merged**